app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -182,15 +184,8 @@ def pull_manifest
     end
 
     def check_blob
```

Comment:
I have a feeling the image size issue is something with this method. We _were_ setting a content length value in the response header that doesn't seem to exist anymore? Qualifier on this advice: I may not know what I'm talking about.

Reply:
Our code actually errors out before we even reach this method, the issue is between podman and puma rather than our code. The MAX_CHUNK_HEADER_SIZE is so small that puma rejects the request that podman sends to us. I'm going to report an issue to puma to see what they say.

Reply:
https://github.com/puma/puma/issues/3362

Reply:
Good to know, thanks! Interesting issue. Are you wanting to keep this on the back burner until they get back to you or push this through under the pretense larger image sizes should be fine in the future?

Reply:
Awesome, thanks!